### ISSUE TITLE
No separate windows after removing `should_launch_new_instance`

### ISSUE THREAD

**Issue by BonfireAtNight** [ahrm/sioyek#424]
Until today I had both `should_launch_new_instance 1` and `should_launch_new_window 1` in my `prefs_user.config`. I've noticed that the documentation says that the former is now considered deprecated, so I removed it.

I set up my window manager (i3) so that new Sioyek windows would appear tabbed within the same workspace. It worked perfectly before the changes. Now, if Sioyek is already running and I open a new file, the new instance (window?) replaces the old. Strangely, the issue persists even after including `should_launch_new_instance 1` again.

I have the feeling that now it doesn't open any new window at all, so that's why i3 isn't doing what was expected? For instance, I've moved the new instance (window?) to a different workspace to see if the one I first opened was underneath, but this doesn't seem to be the case. If I run the `goto_window` command it doesn't list the first-opened file, either.

**Comment by ahrm** [comment]
That is strange. Maybe you can share your `prefs_user.config` here so I can see if I can replicate the results. Also which sioyek version are you using?
Also does this happen on all documents (for example does this happen on the sioyek tutorial PDF?)

**Comment by BonfireAtNight** [comment]
This is my `prefs_user.config`:
```bash
# Size of the UI font
font_size 20
# Size of the GUI elements at the bottom and top of the screen
status_bar_font_size 20

# Synctex
inverse_search_command  nvr --remote-silent %1 -c %2

search_url_g https://www.google.com/search?q=

page_separator_width 1
page_separator_color 0.5 0.5 0.5

should_launch_new_window 1 # I think this is the default now?
# should_launch_new_instance 1 # Deprecated
```
If tested with the exact same combination of files for which it had still been working yesterday (same order, too). I've replaced one of the two documents with the tutorial, still the same effect. I'm on version 1.5.0.

**Comment by ahrm** [comment]
Are you trying opening a new window for the same document (which already has a window)?
If so, we reuse the old window (if you want to create a new window for the same document you have to explicitly pass `--new-window` command line option).

**Comment by ahrm** [comment]
Nevermind, I was able to reproduce the problem with different documents.

**Comment by ahrm** [comment]
Should be fixed in 219204d85dc55bcecc2fae2b2beb4689ebc1dc3d.

By the way, this type of comment:
```
should_launch_new_window 1 # I think this is the default now?
```
is not allowed in sioyek, the only comment that we support is full line comments. So if you use exactly this config ifle, `should_launch_new_window` will not be set properly (I know your actual config file probably doesn't have this comment, I just mentioned it just in case).

**Comment by BonfireAtNight** [comment]
Great, thanks! And once again, thanks for all the great work you are doing here in general!
